trademarks.md

```diff
@@ -0,0 +1,233 @@
+# Trademark Usage Policy
+
+**The following is a draft of the Jupyter Trademark Usage Policy.
+We are actively working on this document.
+Sections that are completed should be considered to be accepted and active.**
+
+The goal of this document is to clarify what is allowed when using the Jupyter trademarks,
```

Comment:
@minrk Perhaps:

This document's goal is to clarify allowed uses of the Jupyter trademarks. We wish to implement a very permissive policy that legally protects the trademarks while also encouraging the community to liberally use the trademarks. This policy will attempt to clarify which uses are explicity allowed without the need for approval: